.travis.yml

```diff
@@ -1,14 +1,17 @@
 os: linux
 sudo: required
-dist: xenial
-services:
-  - xvfb
-  - postgresql
-language: python
+dist: trusty
```

Comment:
why are we going back on the linux version? I believe this is the cause of the build fail